### DISCUSSION TITLE
Wix 4, Visual Studio test discovery.

### DISCUSSION THREAD

**Discussion by gbull122** [wixtoolset/issues#7642] [Questions]
In Visual Studio 2022(17.5.1) I've successfully updated  a solution's msi package and bundle projects from 3.11 to 4.0.1 using HeatWave 1.0.1. I now see in the Visual Studio output for tests, each wix 4.0.1 project is reporting, `Failed to parse target framework ',Version=v0.0' for project 'XXXXXXXX'.`  The solution also contains MSTest projects, 

![image](https://github.com/wixtoolset/issues/assets/3079682/a774116e-3078-4e0a-8832-40c0dcfca7a6)

I tried adding,

`<PropertyGroup>
	<IsTestProject>false</IsTestProject>
</PropertyGroup>
<ItemGroup>
	<ProjectCapability Remove="TestContainer" />
</ItemGroup>`

to each wixproj file in the hope the projects would be ignored by VS test discovery but doesn't help. All wix projects build and run but have I missed something in upgrading to 4.0.1?

**Comment by robmen** [comment]
Did you get to the bottom of this issue?

**Reply by james1301** [reply]
@robmen no not managed to get to the bottom of it. Any help?
